src/agent/src/config.rs

```diff
@@ -24,6 +25,8 @@ const LOG_VPORT_OPTION: &str = "agent.log_vport";
 const CONTAINER_PIPE_SIZE_OPTION: &str = "agent.container_pipe_size";
 const UNIFIED_CGROUP_HIERARCHY_OPTION: &str = "agent.unified_cgroup_hierarchy";
 const CONFIG_FILE: &str = "agent.config_file";
+const HTTPS_PROXY: &str = "agent.https_proxy";
+const NO_PROXY: &str = "agent.no_proxy";
```

Comment:
Is there a good place to add some description of these new configs? The `no_proxy` option in particular isn't totally intuitive to me.

Reply:
Sure. Thanks for review. I'll add comments here.

Reply:
Fixed.

Reply:
I had this https://github.com/kata-containers/kata-containers/pull/4016 to update the agent configs on the README but I gave up due the lack of time. I can revive that PR and update with the options introduced here. In a follow on PR... what do you think @jodh-intel @fitzthum ?

Reply:
@wainersm - You are in luck: once #9110 lands, you could just update the table.

Reply:
Done.